bigbluebutton-tests/playwright/settings/language.js

```diff
@@ -1,20 +1,44 @@
 const Page = require('../core/page');
-const { openSettings } = require('./util');
+const { openSettings, getLocaleValues } = require('./util');
 const e = require('../core/elements');
+const { checkNotificationText } = require('../notifications/util');
 
 class Language extends Page {
   constructor(browser, page) {
     super(browser, page);
   }
 
   async test() {
-    for(const locale of e.locales) {
+    const selectedKeysBySelector = {
+      [e.messageTitle]: 'app.userList.messagesTitle',
+      [e.notesTitle]: 'app.userList.notesTitle',
+      [e.userList]: 'app.navBar.userListToggleBtnLabel',
+      [e.hidePublicChat]: 'app.chat.titlePublic',
+      [e.sendButton]: 'app.chat.submitLabel',
+      [e.actions]: 'app.actionsBar.actionsDropdown.actionsLabel',
+      [e.joinAudio]: 'app.audio.joinAudio',
+      [e.joinVideo]: 'app.video.joinVideo',
+      [e.startScreenSharing]: 'app.actionsBar.actionsDropdown.desktopShareLabel',
+      [e.minimizePresentation]: 'app.actionsBar.actionsDropdown.minimizePresentationLabel',
+      [e.raiseHandBtn]: 'app.actionsBar.emojiMenu.raiseHandLabel',
```

Comment:
As a side note: we should probably stick to some naming convention for the element selectors (for example, all button selectors should end with either `Button` or `Btn`, not both). We can address this in some future PR.